src/AppBundle/Entity/Report/Report.php

```diff
@@ -388,12 +393,18 @@ public static function getSectionsSettings()
     /**
      * @var Checklist
      *
+     * // TODO "report" group is used by deputy side each time a report is loaded. Better to use a new one like report-checklist
+     *
      * @JMS\Groups({"report"})
```

Comment:
I thought this was report-checklist. May have been reverted at some time?

Reply:
don't know. It probably contributed to slow down the dashboard as it loaded lots of data, but difficult to say, the object hydration was taking too much data anyway. Not a problem anymore with the current solution, the dashboard was teh only point where we load more than 2 reports, and it now just runs 4 simple queries